$NetBSD: patch-Cargo.toml,v 1.4 2021/05/25 16:15:54 pin Exp $

Don't pull the vendored version of mlua, use native luajit

--- Cargo.toml.orig	2021-05-25 10:54:37.000000000 +0000
+++ Cargo.toml
@@ -28,7 +28,7 @@ lazy_static = "1.4.0"
 indexmap = { version = "1.6.2", features = ["serde"] }
 natord = "1.0.9"
 humansize = "1.1.0"
-mlua = { version = "0.5.4", features = ["luajit", "vendored", "serialize", "send"] }
+mlua = { version = "0.5.4", features = ["luajit", "serialize", "send"] }
 ansi-to-tui = "0.2.0"
 libc = "0.2.94"
 
